cmd/wego/gitops/cmd.go

```diff
@@ -72,7 +73,8 @@ func init() {
 
 func installRunCmd(cmd *cobra.Command, args []string) error {
 	cliRunner := &runner.CLIRunner{}
-	fluxClient := flux.New(cliRunner)
+	osysClient := osys.New()
```

Comment:
small nit. I would suggest to use this pattern also in `cmd/wego/flux/cmd.go` to be consistent.

Reply:
Sure, okay. I only originally didn't because the osysclient was used in two places but that's no big deal. :+1: